docs/CONFIG.md

```diff
@@ -9243,6 +9308,33 @@ TipCapMinimum is the minimum gas tip to use when submitting transactions to the
 
 (Only applies to EIP-1559 transactions)
 
+## EVM.GasEstimator.DAOracle
+```toml
+[EVM.GasEstimator.DAOracle]
+OracleType = 'OPOracle' # Example
+OracleAddress = '0x420000000000000000000000000000000000000F' # Example
+CustomGasPriceAPICalldata = '' # Default
+```
+
+
+### OracleType
+```toml
+OracleType = 'OPOracle' # Example
```

Comment:
OP`Oracle` is redundant, can we just use straight forward names, If anything I'd prefer similar name as the chaintypes for consistency

Reply:
let's do `OPStackOracle` so it's easier to understand for folks not familiar; we want this to be more generic than chaintypes, Oracle types should work with DA families, e.g `optimismBedrock` is a bad example, technically bedrock is a thing of the past

Reply:
I think @simsonraj 's point is that the `...Oracle` suffix is the redundant portion. What about just `OPStack` @matYang ? And for arbitrum/zksync, would you prefer `ArbitrumStack` and `ZKSyncStack` or just `arbitrum` and `zksync`?

Reply:
Got it, yeh makes sense. OPStack is the actual term used by OP family chains, I dont think `ArbitrumStack` or `ZKSyncStack` is a thing. Arbitrum Orbit family might be on the table but I haven't looked into its DA solution, out of scope of this PR for sure.

Reply:
renamed to `opstack`